src/Components/PlaylistCard.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { useData } from "../Context/DataProvider";
+import {
+  CheckIfVideoExistsInList,
+  formatDate,
+  formatNumber,
+  formatString
+} from "../utils";
+import { Link } from "react-router-dom";
+import { HiDotsVertical } from "react-icons/hi";
+import { MdWatchLater, MdRemoveCircle } from "react-icons/md";
+export function PlaylistCard({ videoId, playlistId }) {
+  const {
+    state: { videos, watchLater },
+    addToHistory,
+    addToWatchLater,
+    dispatch
+  } = useData();
+  function getPlaylist() {
+    return videos.find((video) => video.id === videoId);
+  }
+  const video = getPlaylist();
+  function removeFromPlaylist() {
+    dispatch({
+      type: "SET_VIDEO_ID_TO_BE_ADDED_TO_PLAYLIST",
+      payload: videoId
+    });
+    dispatch({
+      type: "TOGGLE_ADD_TO_PLAYLIST",
+      payload: playlistId
+    });
+  }
+  return (
+    <div>
+      {!video.length > 0 ? (
+        <div>
+          <div
+            style={{ width: "230px" }}
+            key={video.id}
+            className=" vertical-card link-btn"
+          >
+            <Link
+              to={`/video/${video.id}`}
+              className="link-btn"
+              onClick={() => addToHistory(video)}
+            >
+              <img
+                style={{ width: "230px" }}
+                alt="thumbnail"
+                src={video.thumbnail}
+              />
+            </Link>
+
+            <div className="smallest-text">
+              <div>
+                <div className="flex space-between">
+                  <Link
+                    className="bold-text link-btn"
+                    to={`/video/${video.id}`}
+                    onClick={() => addToHistory(video)}
+                  >
+                    {formatString(video.title)}
+                  </Link>
+                  <div className="tooltip pointer">
+                    <HiDotsVertical size="1rem" color="grey" />
+                    <div
+                      style={{ color: "grey" }}
+                      className="tooltiptext card-shadow light-border"
+                    >
+                      {!CheckIfVideoExistsInList(watchLater, video.id) ? (
```

Comment:
In react, functional components should have capitalize first letter. Everything else should be camel case as per convention. I believe it is a pure function right? therefore it should be camelcase.

Reply:
I have made it right now.